File: Net5.AspNet.Workshop.Client/ClientApp/src/app/services/data/workshop/change-enrollment-states-filter.ts
import { Workshop } from "../../../models/workshop";

export class ChangeEnrollmentStatesFilter {
  constructor() {
    this.EnrollmentIds = new Array<number>();
    this.EnrollmentStateId = 0;    
  }

  public EnrollmentIds: Array<number>;
  public EnrollmentStateId: number;
}

File: Net5.AspNet.Workshop.Workshop.Api/Infrastructure/FilterParameters/ChangeEnrollmentStatesFilter.cs
using Net5.AspNet.Workshop.Infrastructure.Dtos;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Net5.AspNet.Workshop.Workshop.Api.Infrastructure.FilterParameters
{
    public class ChangeEnrollmentStatesFilter
    {
        public ChangeEnrollmentStatesFilter()
        {
            EnrollmentIds = new List<int>();            
        }
        public List<int> EnrollmentIds { get; set; }
        public int enrollmentStateId { get; set; }        
    }
}
